dist/exit-from-dialog.js

```diff
@@ -404,6 +412,43 @@ function _getExitParams(parameters) {
 		transition_effect,
 	};
 }
+
+addDualDialogTag('exitHere', function (environment, parameters) {
+	var exit = _getExitHereParams(parameters);
+	if (!exit) {
+		return;
+	}
+	bitsy.movePlayerThroughExit(exit);
+});
+
+function _getExitHereParams(parameters) {
+	var params = parameters[0].split(',');
+	var roomName = params[0];
+	var transition_effect = params[1];
+	var room = getRoom(roomName).id
+	
+	var x = player().x;
+	var y = player().y;
+
+	if (!roomName || x === undefined || y === undefined) {
```

Comment:
since this is using the player position, the checks on position are unnecessary:

```suggestion
	if (!roomName) {
```

Reply:
I'm pretty sure the position will still be undefined if you tried to use _exitHere_ in the title dialog, but I have to double check.

Reply:
Checked again and it worked fine, so removed the position check in the latest commit.~